phpspec.yml.ci

```diff
@@ -0,0 +1,10 @@
+suites:
+  data_import_suite:
+    namespace: Ddeboer\DataImport
+    psr4_prefix: Ddeboer\DataImport
+extensions:
+  - PhpSpec\Extension\CodeCoverageExtension
+formatter.name: pretty
+code_coverage:
```

Comment:
I prefer not to use Behat. Behat can be very useful for client-focused web tests, but tests for a library are different: they should make sense to developers. Moreover, Behat adds considerable overhead in terms of writing and maintaining step definitions for Gherkin. So let’s keep PHPUnit for integration tests.